game/scripts/npc/abilities/ember_spirit_flame_guard.txt

```diff
@@ -56,7 +56,7 @@
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_per_second"                               "30 40 50 60 120 240"
+        "damage_per_second"                               "30 40 50 60 180 300"
```

Comment:
I am fine with talent buff but this seems too much. I would just keep at normal and with talent it would be 360 max (which is equal to radiance level 5)

Reply:
I buffed every magic aspect of Ember in recent patches. So revert this.

Reply:
This hero is really bad. This hero can't even tickle enemy healthbars. A 60 (OAA levels) + 60 (Talent) DPS buff to a dispellable spell won't be OP.

Reply:
Put it at 180/360. And set the Flame Guard talent to +180. I am sure it will not be balanced but lets see.

Reply:
Ive buffed the flame guard damage at level 6.

Reply:
I made a mistake when buffing flame guard damage talent. I buffed the wrong talents. I wanted flame Guard damage to be +120 DPS not +200. I wanted sleight of fist to be +200 but I reduced it to +160.